### DISCUSSION TITLE
lc/997/

### DISCUSSION THREAD

**Discussion by giscus** [doocs/leetcode#2753] [Announcements]
# lc/997/

多种编程语言实现 LeetCode、《剑指 Offer（第 2 版）》、《程序员面试金典（第 6 版）》题解

https://leetcode.doocs.org/lc/997/

**Comment by lazyJLBL** [comment]
这道题有没有建图然后做的题解呀

**Reply by yanglbme** [reply]
没有，计数的方法是最简单的噢

**Reply by lazyJLBL** [reply]
好吧
